plugins/DevicesDetection/Columns/DeviceBrand.php

```diff
@@ -60,4 +60,15 @@ public function onNewVisit(Request $request, Visitor $visitor, $action)
 
         return $parser->getBrand();
     }
+
+    /**
+     * @param Request $request
+     * @param Visitor $visitor
+     * @param Action|null $action
+     * @return mixed
+     */
+    public function onAnyGoalConversion(Request $request, Visitor $visitor, $action)
```

Comment:
@mattab this will add columns to `log_conversion`. So for the other dimensions. I presume it's maybe not as much of a major update to add these columns or shall we merge into Piwik 3?